src/Constants/SocketEvents.js

```diff
@@ -9,6 +9,8 @@ const SOCKET_EVENTS = {
   DISPLAY: 'display',
   UPDATE_PLAYLIST: 'update-playlist',
   PLAYLIST_UPDATED: 'playlist-updated',
+  KICK_USER: 'kick-user',
+  KICKED_USER: 'kicked-user',
```

Comment:
```suggestion
  USER_KICKED: 'user-kicked',
```
"kicked user" banlanmış kullanıcı gibi anlaşılırken "user kicked" kullanıcı banlandı olarak anlaşılacağını düşünüyorum. İsimde değişiklik yapabiliriz.